conjur/credentials_from_file.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+"""
+CredentialsFromFile module
+
+This module holds the logic for writing user-specific credentials
+to a netrc file on the user's machine when they login
+"""
+
+# Builtins
+import logging
+import netrc
+import os
+import stat
+
+# Internals
+from conjur.constants import DEFAULT_NETRC_FILE
+
+class CredentialsFromFile:
+    """
+    CredentialsFromFile
+
+    This class holds logic when credentials are kept in the netrc
+    """
+    @classmethod
```

Comment:
In OOP, you can use object to handle this, instead of static class methods.
You could pass the credentials file path as ctor parameter, instead of referring to the const every time.
Its default value could be the const, so you won't have to supply it from outside.

Reply:
> In OOP, you can use object to handle this, instead of static class methods.

Not sure what you mean here

> Its default value could be the const, so you won't have to supply it from outside.

In terms of value. What would this give me? Since I can just change the default netrc path in constants and that's it

Reply:
In terms of value, this class should not rely on const paths, as its purpose is reading and writing the credentials to a file.
The actual path should not impact its behaviour, so it shouldn't depend on a const for that.
It should be a parameter for the object.